portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -33,9 +33,41 @@ definition {
 	@ignore = "true"
 	@priority = "5"
 	test AnonymousIndividualsDoNotCountAsIndividualsForUsage {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
 
-		// TODO LRAC-8147 Usage#AnonymousIndividualsDoNotCountAsIndividualsForUsage pending implementation
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		ACUtils.launchDXP();
+
+		JSONUser.addUser(
```

Comment:
@AfonsoHSantos, you need to login with this user you created

Reply:
I think you'd better create a `Site Name` page and connect the AC to that page. Because to make sure that only the **TEST TEST** login when you go back to DXP is not affecting these numbers

Reply:
fixed